diff --git a/node_modules/readdir-glob/node_modules/minimatch/minimatch.js b/node_modules/readdir-glob/node_modules/minimatch/minimatch.js
index 8545b3a..c8d9d8e 100644
--- a/node_modules/readdir-glob/node_modules/minimatch/minimatch.js
+++ b/node_modules/readdir-glob/node_modules/minimatch/minimatch.js
@@ -16,7 +16,7 @@ minimatch.sep = path.sep
 
 const GLOBSTAR = Symbol('globstar **')
 minimatch.GLOBSTAR = GLOBSTAR
-const expand = require('brace-expansion')
+const expand = require('brace-expansion').expand
 
 const plTypes = {
   '!': { open: '(?:(?!(?:', close: '))[^/]*?)'},
